### PR TITLE
Refactor/fix various things

### LINKED ISSUE #19: Cypress Testing + various refactors
- Refactor 5 star tests
- Add 's' after HTTP to all stubs in test files
- As a user, if I hit either arrow in the nav bar I should see the rest of the filter types 
- As a user, I should see a header that describes the type of products I'm viewing ex.(cy.get('h2').contains('Featured Items'))
- As a user, I should see a helpful message if a section has no items

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Added a helpful message if no saved items are available
- Fixed URLs to properly stub requests
- Refactored featured products tests
- Fixed alt name for right arrow button to improve accessibility 
- Closes #19 

Testing Notes/Extras: 
All tests are passing. Tests created for the following user stories: 
> - As a user, if I hit either arrow in the nav bar I should see the rest of the filter types
> - As a user, I should see a header that describes the type of products I'm viewing 
> - As a user, I should see a helpful message if a section has no items

Did this break anything?

- [x]  No
- [ ]  Yes

Type of change

- [x]  New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Creating error component

### CODE REVIEW

File: cypress/e2e/featured-products.cy.js [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
-    .checkRating('Precision Brow Pencil')
-    .checkRating('Sculpt & Highlight Brow Contour')
-  })
-
-  it('Should only contain unique products', () => {
-    cy.checkUniquity('Serum Foundation')
-    .checkUniquity('Coverage Foundation')
-    .checkUniquity('Precision Brow Pencil')
-    .checkUniquity('Sculpt & Highlight Brow Contour')
+  it('Should display 4 unique product cards with ratings strictly equal to 5', () => {
```
Comment (paulina-isabel) [inline cypress/e2e/featured-products.cy.js:17]: Good job condensing these into one test.

File: src/components/CategoryContainer/CategoryContainer.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const CategoryContainer = ({selectedCategory, updateCategory}: CategoryContainer
       <section className="category-container">
         {categoryElements}
       </section>
-      <button onClick={() => scroll(200)} className='right-scroll-btn'><img src={forward} alt='back button'/></button>
+      <button onClick={() => scroll(200)} className='right-scroll-btn'><img src={forward} alt='forward button'/></button>
```
Comment (paulina-isabel) [inline src/components/CategoryContainer/CategoryContainer.tsx:30]: Good catch.

**Review by paulina-isabel — APPROVED** [review]
Great improvements.

**Review by lauraguerra1 — APPROVED** [review]
great job on this!

**Pull request merged**